api/app/graphql/spree/types/order_type.rb

```diff
@@ -0,0 +1,14 @@
+module Spree
+  class Types::OrderType < ::GraphQL::Schema::Object
+    field :id, ID, null: false do
+      # argument :action_type, GraphQL::Types::String, default_value: :read, required: false
+      # resolve(obj, args, ctx) { binding.pry }
+    end
+    # You can only see the details on a `Friendship`
+    # if you're one of the people involved in it.
+    def self.authorized?(object, context)
+      binding.pry
```

Comment:
@gingray remove `binding.pry` from here pls ;)

Reply:
it still work in progress status dirty commits )))